Zotero/Scenes/Detail/Annotation Popover/Views/HighlightEditCell.swift

```diff
@@ -10,49 +10,71 @@ import UIKit
+            textView.textContainerInset = UIEdgeInsets()
+            textView.textContainer.lineFragmentPadding = 0
+            textView.delegate = self
+            textView.isScrollEnabled = false
+            textView.translatesAutoresizingMaskIntoConstraints = false
+            contentView.addSubview(textView)
+            self.textView = textView
+
+            NSLayoutConstraint.activate([
+                lineView.topAnchor.constraint(equalTo: textView.topAnchor),
+                lineView.bottomAnchor.constraint(equalTo: textView.bottomAnchor),
+                lineView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+                lineView.widthAnchor.constraint(equalToConstant: 3),
+                textView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 10),
+                textView.leadingAnchor.constraint(equalTo: lineView.trailingAnchor, constant: 8),
+                contentView.bottomAnchor.constraint(equalTo: textView.bottomAnchor, constant: 10),
+                contentView.trailingAnchor.constraint(equalTo: textView.trailingAnchor, constant: 16)
+            ])
         }
     }
 
-    var caretRect: CGRect? {
-        guard self.textView.isFirstResponder, let selectedPosition = self.textView.selectedTextRange?.start else { return nil }
-        return self.textView.caretRect(for: selectedPosition)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-
-        self.textView.textContainerInset = UIEdgeInsets()
-        self.textView.textContainer.lineFragmentPadding = 0
-        self.textView.delegate = self
-        self.textView.isScrollEnabled = false
+    var caretRect: CGRect? {
+        guard let textView, textView.isFirstResponder, let selectedPosition = textView.selectedTextRange?.start else { return nil }
+        return textView.caretRect(for: selectedPosition)
     }
 
-    func setup(with text: String, color: String) {
+    func setup(with text: NSAttributedString, color: String) {
         let paragraphStyle = NSMutableParagraphStyle()
         paragraphStyle.minimumLineHeight = AnnotationPopoverLayout.annotationLayout.lineHeight
         paragraphStyle.maximumLineHeight = AnnotationPopoverLayout.annotationLayout.lineHeight
-        let attributedText = NSAttributedString(string: text, attributes: [.font: AnnotationPopoverLayout.annotationLayout.font,
-                                                                           .foregroundColor: Asset.Colors.annotationText.color,
-                                                                           .paragraphStyle: paragraphStyle])
+        let attributedText = NSMutableAttributedString(attributedString: text)
+        attributedText.addAttributes([.foregroundColor: Asset.Colors.annotationText.color, .paragraphStyle: paragraphStyle], range: .init(location: 0, length: attributedText.length))
```

Comment:
Same as above, isn't passing the same array in initializer simpler?

Reply:
`text` has also changed to an attributed string.